lib/service/database/table.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:sqflite/sqflite.dart';
+
+import 'database_config.dart';
+
+@immutable
+class Table {
+  final String tableName;
+  final String idCol;
+  final String createdCol;
+  final String updatedCol;
+  final String isDeletedCol;
+  final String remarkCol;
+  final String referenceCol;
+
+  const Table({
+    required this.tableName,
```

Comment:
@irfanSilkron explain diff between required dgn takde required?

Reply:
kalau ada required, bila initialize Table punya class, wajib pass tableName as parameter. Kala takde tak wajib pass as parameter, instead it will use default value.